setup.sh

```diff
@@ -18,16 +19,16 @@ declare -r app_name='kw'
 ## Following are the install paths
 ##
 # Paths used during the installation process
-declare -r kwbinpath="$HOME/.local/bin/$app_name"
-declare -r binpath="$HOME/.local/bin"
-declare -r libdir="$HOME/.local/lib/$app_name"
-declare -r sharedir="${XDG_DATA_HOME:-"$HOME/.local/share"}/$app_name"
-declare -r docdir="$sharedir/doc"
-declare -r mandir="$sharedir/man"
-declare -r sounddir="$sharedir/sound"
-declare -r databasedir="$sharedir/database"
+declare kwbinpath="$HOME/.local/bin/$app_name"
```

Comment:
Why remove `-r`? It looks unsafe...

Reply:
Because I need to change the value of the variable if it is system-wide installation. It will be at `/usr/bin/kw`. It can't be readonly, therefore.